playbooks/packstack.yaml

```diff
@@ -287,32 +286,59 @@
     - name: Fix endpoints in keystone database
       block:
         - name: Look for HTTPS 5000/v2.0 url
-          shell: mysql keystone -s -N -e 'select url from endpoint where url LIKE "http://%5000/v2.0"' | wc -l
+          shell: >
```

Comment:
AVOID SHELL ! IT IS A WEAK MAN's CRUTCH !
Use something from `http://docs.ansible.com/ansible/latest/modules/list_of_database_modules.html#mysql`

Reply:
I do not see a way with this modules to easily interact with the database apart using `mysl_db` with `state: import` and a .sql file.
I think that in fact it would be better to use the native os_ modules to edit the endpoints, could you please add a bug for this, as it will have to make clear what the endpoints could be?